packer/files/macos/bashrc

```diff
@@ -1,5 +1,5 @@
 export PATH=/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:$PATH
-export PATH=/opt/local/bin:$PATH
+export PATH=/opt/homebrew/bin:/opt/local/bin:$PATH
```

Comment:
Since this `bashrc` works for legacy `jenkins-agent-macos12-x64.json` , should we move this to legacy folder as well?

Reply:
This also works for the new macos13 that is why I keep it here.

Thanks.